### ISSUE TITLE
Unable to build a website locally

### ISSUE THREAD

**Issue by worldbeater** [reactiveui/website#151]
Not sure what is causing this, but now I am constantly getting this error:
```
Error: The assembly 'Cake.Wyam, Version=1.2.0.0, Culture=neutral, PublicKeyToken=null'
is referencing an older version of Cake.Core (0.22.2).
This assembly must reference at least Cake.Core version 0.26.0.
Another option is to downgrade Cake to an earlier version.
It's not recommended, but you can explicitly opt out of assembly verification
by configuring the Skip Verification setting to true
(i.e. command line parameter "--settings_skipverification=true",
environment variable "CAKE_SETTINGS_SKIPVERIFICATION=true",
read more about configuration at https://cakebuild.net/docs/fundamentals/configuration)
```

**Steps to reproduce:**
1. Clone the repo
2. Try to build it following the instructions from README 
```
.\build.cmd -Target preview
```

So I'm unable to contribute :)

**Comment by daveaglick** [comment]
The underlying problem looks like Wyam is pinned to 1.2.0 which was released back in January. There's been a few breaking changes in Cake over that time so when the Cake version used by the website build recently got updated, it fell out of sync with the older version of Wyam (newer version of Wyam support the newer version of Cake).

The fix in #152 should resolve the problem short-term since I don't think the Cake.Wyam addin is doing anything exotic that would break on a newer version of Cake. A better long-term fix is to update Wyam to the latest (1.5.0 was released yesterday). I'll take a look at doing PR for that update.

**Comment by glennawatson** [comment]
Rodney tried updating to 1.4 something very recently and we were getting exceptions in the blog generation so decided to keep it at 1.2 until we could work it out.

**Comment by RLittlesII** [comment]
The problem is this PR https://github.com/reactiveui/website/pull/144

I was unable to build locally because I have global cake config and am setup for global dependencies.

When I made that change I was trying to avoid making the change to the bootstrapper.  Hind  sight, that was the right place to fix the problem.

With the Cake version bump I elected to stay with Wyam 1.2 because 1.4 blog generation broke.

So we should take the --settings_skipverification and move it to the bootstrappers so this issue goes away.
